webapp/src/pages/home/Home.js

```diff
@@ -53,7 +53,8 @@ class Home extends Component {
               return (
                 <td className="Cell">
                   {
-                    this.state.rows[rowIndex][fieldNames[cellIndex]] === "Apply now" ?
+                    this.state.rows[rowIndex][fieldNames[cellIndex]] === "Apply now" || 
+                      this.state.rows[rowIndex][fieldNames[cellIndex]] === "Continue application" ?
                       <NavLink to="/applicationForm">Apply now</NavLink> :
```

Comment:
How about changing the "Apply Now" text to {this.state.rows[rowIndex][fieldNames[cellIndex]]}  ?

Reply:
oh wait. yeah, sorry, stupid, didn't pay attention